nsdb-kafka-connect/src/main/scala/io/radicalbit/nsdb/connector/kafka/sink/NsdbSinkWriter.scala

```diff
@@ -173,21 +179,30 @@ object NsdbSinkWriter {
     var bit: Bit = Db(valuesMap(dbField).toString).namespace(valuesMap(namespaceField).toString).bit(parsedKcql.metric)
 
     val timestampField = aliasMap("timestamp")
-    val valueField     = aliasMap("value")
+    val valueFieldOpt  = aliasMap.get("value")
 
     valuesMap.get(timestampField) match {
-      case Some(t: Long) => bit = bit.timestamp(t)
-      case Some(v)       => sys.error(s"Type ${v.getClass} is not supported for timestamp field")
-      case None          => bit = bit.timestamp(System.currentTimeMillis())
+      case Some(t: Long)                          => bit = bit.timestamp(t)
+      case Some(v)                                => sys.error(s"Type ${v.getClass} is not supported for timestamp field")
+      case None if timestampField == "sys_time()" => bit = bit.timestamp(System.currentTimeMillis())
```

Comment:
So that it turns out that `"sys_time()"` is a string at the end instead of a placeholder? In this case, why don't we use  something more comprehnsive (e.g. `current_time`, `processing_time`, `now`)?

Reply:
Good point. `sys_time()` is the default value proposed by the library. There is no reason why we could not adopt another keyword. Let's vote: 

* now
* processing_time
* current_time
* none of the above
* all the above (with or without brackets)

My 2 cents... now with or without brackets